src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-pypemicro.py

```diff
@@ -0,0 +1,39 @@
+# ------------------------------------------------------------------
+# Copyright (c) 2022 PyInstaller Development Team.
+#
+# This file is distributed under the terms of the GNU General Public
+# License (version 2.0 or later).
+#
+# The full license is available in LICENSE.GPL.txt, distributed with
+# this software.
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+# ------------------------------------------------------------------
+
+# Hook for the pypemicro module: https://github.com/nxpmicro/pypemicro
+
+import os
+from PyInstaller.utils.hooks import remove_prefix, get_package_paths, is_module_satisfies
+from PyInstaller.log import logger
+from PyInstaller.compat import is_darwin
+
+binaries = list()
+if is_module_satisfies('pyinstaller < 5.0'):
```

Comment:
This is probably a matter of personal preference, but I'd rather see `is_module_satisfies('pyinstaller >= 5.0')` here (and consequently warning in the else branch).

Reply:
Oh, I missed this one, will do another (squashed) commit